etc/config.xml

```diff
@@ -11,7 +11,7 @@
             <settings>
                 <allowed_modules><![CDATA[]]></allowed_modules>
                 <rule_modules>deniedByDefault</rule_modules>
-                <allowed_tabs>ddg_automation</allowed_tabs>
+                <allowed_tabs>yotpo,ddg_automation</allowed_tabs>
```

Comment:
Any reason to allow Yotpo? We have received a request from some people to move this to the Extension section by default.

Reply:
you are right, this one is something I missed in the merge conflict, my bad

Reply:
No problem. Thanks.